api/lib/domain/events/handle-certification-scoring.js

```diff
@@ -1,15 +1,15 @@
+const CertificationScoringCompleted = require('./CertificationScoringCompleted.js');
 const AssessmentResult = require('../models/AssessmentResult');
-const Badge = require('../models/Badge');
-const CertificationPartnerAcquisition = require('../models/CertificationPartnerAcquisition');
 const CompetenceMark = require('../models/CompetenceMark');
 const Promise = require('bluebird');
```

Comment:
```suggestion
const bluebird = require('bluebird');
```

(on n'écrase pas l'objet `Promise` natif)